### ISSUE TITLE
React 16.14.0

### ISSUE THREAD

**Issue by t2ca** [system-ui/theme-ui#1212]
I cannot build my site without getting errors ever since the release of React 16.14 which has added support for the new JSX transform.  This is the same error that we had when upgrading to React17 https://github.com/system-ui/theme-ui/issues/1160

edit: I was able to get it working again after downgrading `gatsby-plugin-theme-ui` back to `v0.3.0`

edit2: After going back to `v0.3.0`, I was able to upgrade to `0.4.0-rc.5` and it would keep working without any errors until I upgraded some other packages and the error would show up again. So each time i see the error, I will just install `v0.3.0` followed by `0.4.0-rc.5`.

edit3: I forgot to mention that i also had to change react and react-dom to `16.13.1`

**Comment by atanasster** [comment]
I believe this also waits on the emotion update - maybe close this issue as dupe?

**Comment by lachlanjc** [comment]
@hasparus Duplicate of Emotion 11/React 17 Runtime support, we can close

**Comment by chenrui333** [comment]
@lachlanjc, 👋  can you attach the link for Emotion 11/React 17 Runtime support (I am also looking for the React 17 compatibility support)? Thanks!

**Comment by lachlanjc** [comment]
That would be #1237!

**Comment by chenrui333** [comment]
Ah, so this is already done. That is awesome! 🎉

**Comment by hasparus** [comment]
@chenrui333 it's not yet in stable release.

BTW React 17 was always working -- it was just the automatic runtime support and people were surprised by the change of defaults in CRA. 

You can still use the classic JSX runtime (if you can configure your Babel config) with React 17, so even the oldest versions of Theme UI should work.
